### ISSUE TITLE
mismatch since addition of pyproject.toml support

### ISSUE THREAD

**Issue by jugmac00** [mgedmin/check-python-versions#41]
```
check-python-versions....................................................Failed
- hook id: check-python-versions
- exit code: 1

setup.py says:                   3.7, 3.8, 3.9, 3.10, 3.11
- python_requires says:          3.7, 3.8, 3.9, 3.10, 3.11
pyproject.toml says:             (empty)
tox.ini says:                    3.7, 3.8, 3.9, 3.10, 3.11
.github/workflows/main.yml says: 3.7, 3.8, 3.9, 3.10, 3.11

mismatch!
```

This regression happened after updating ...
[github.com/mgedmin/check-python-versions: 0.20.0 → 0.21.1](https://github.com/mgedmin/check-python-versions/compare/0.20.0...0.21.1)

pre-commit run:
https://results.pre-commit.ci/run/github/275214686/1676310664.enB_WFtiQ6OGMh2mhmpNcg

Looks like the issue is my pyproject.toml

```
[build-system]
requires = ["setuptools>=40.6.0", "wheel"]
build-backend = "setuptools.build_meta"
```

**Comment by mgedmin** [comment]
Should be fixed by dbb3306a0842451bbee19868484f25c856b0b7fe.

**Comment by mgedmin** [comment]
I've released check-python-versions 0.21.2 with the fix.

I apologize for the inconvenience!

**Comment by jugmac00** [comment]
Thanks for the fix! And for providing all these awesome tools!
